h/templates/client/excerpt.html

```diff
@@ -0,0 +1,18 @@
+<div ng-transclude ng-if="!enabled()"></div>
+<div ng-if="enabled()">
+  <div class="excerpt"
```

Comment:
You know, I'd like to, but I don't think I can. I'm not entirely sure if this is because `ngIf` is a transcluding directive as well, but it simply doesn't work, and the contents of the directive are never transcluded.